client/my-sites/checkout/src/components/item-variation-picker/util.ts

```diff
@@ -81,3 +81,22 @@ export function getItemVariantDiscountPercentage(
 
 	return discountPercentage;
 }
+
+export function getItemVariantDiscount(
+	variant: WPCOMProductVariant,
+	compareTo?: WPCOMProductVariant
+): number {
+	const compareToPriceForVariantTerm = getItemVariantCompareToPrice( variant, compareTo );
+
+	// Ignore intro discount if it is a 1 month only discount
+	const variantPrice =
```

Comment:
What's a best way to test with a 1 month introductory offer? What other test cases would be good to check?

Reply:
I'm not sure how. These changes were copied over from the `getItemVariantDiscountPercentage` to return an absolute value instead of a percentage.
This does create a duplicate code, so I decided to refactor the original and return the absolute value, if specified.